english/amnesia/amnesia.md

```diff
@@ -3,7 +3,8 @@ skin creep. I can't get no sleep.
 I try some tea, it gets me nowhere.
 It's probably because the marshmellows where filled with horse hooves
 It's definitely throwing off my groove:(
-Then I open my mac and log on to udacity :)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.